Form/Type/DynamicFormType.php

```diff
@@ -42,10 +48,12 @@ class DynamicFormType extends AbstractType
      */
     public function __construct(
         FormFieldTypePool $typePool,
-        Checksum $checksum
+        Checksum $checksum,
+        ?string $honeyPotField = null
```

Comment:
This typehint should also be removed and `string|null $honeyPotField` added to the construct function